### ISSUE TITLE
Exporting the ES template to the file is not working

### ISSUE THREAD

**Issue by WayneLee5991** [elasticsearch-dump/elasticsearch-dump#735]
**Context**
In order to help us troubleshoot issues with this project, you **must** provide the following details:

- ElasticDump version：6.33.4
- Elasticsearch version：7.3.2
- Node.js version：12.19.0
- Full command you are having issue with (with inputs and outputs)
elasticdump --input=http://user:pass@host:9200/template-filter --output=./template.json --type=template
- A data-set you can share publicly which reproduces the problem

**Describe the bug**
there is no data in the file 'template.json', only show ‘{}’.
but when I use the command 'elasticdump --input=http://user:pass@host:9200/template_name --output=./template_name.json --type=template' to export the template, there is some data in the file 'template_name.json'

**Current behavior**
Describe what's happening now.

**Comment by Lansoweb** [comment]
Do you have a template named 'template-filter'? If not, that's correct saving an empty file.

If you need to export all templates, with the '*':
```bash
elasticdump --input="http://user:pass@host:9200/*" --output=./template.json --type=template
```

**Comment by ferronrsmith** [comment]
Sounds like the `template-filter` template doesn't exist. You can try dumping all templates to see the name.

The command above should be fine (omitting the path also works)

**Comment by WayneLee5991** [comment]
> Do you have a template named 'template-filter'? If not, that's correct saving an empty file.
> 
> If you need to export all templates, with the '*':
> 
> ```shell
> elasticdump --input="http://user:pass@host:9200/*" --output=./template.json --type=template
> ```
become enlightened at once.THANKS！！

**Comment by WayneLee5991** [comment]
> Do you have a template named 'template-filter'? If not, that's correct saving an empty file.
> 
> If you need to export all templates, with the '*':
> 
> ```shell
> elasticdump --input="http://user:pass@host:9200/*" --output=./template.json --type=template
> ```

How to filter out the system indices？like '.kibana', I do not want to export this indices

**Comment by ferronrsmith** [comment]
This feature is available in **elasticdump 6.37.0** and is enabled by default
